fixtures/react_native_windows_e2e_app/windows/RNWTestApp.sln

```diff
@@ -0,0 +1,140 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29324.140
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RNWTestApp", "RNWTestApp\RNWTestApp.csproj", "{72EE9031-61F5-4539-AEE8-F3AD644507B5}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Folly", "..\node_modules\react-native-windows\Folly\Folly.vcxproj", "{A990658C-CE31-4BCC-976F-0FC6B1AF693D}"
```

Comment:
I'd recommend changing these relative paths to $SolutionDir) prefixed paths for less headaches in the future.